File: benchmarks/go/strings.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	n := 100000
	token := "ab"
	var b strings.Builder
	for i := 0; i < n; i++ {
		b.WriteString(token)
	}
	s := b.String()
	slen := len(s)
	var checksum int64
	for _, c := range []byte(s) {
		checksum += int64(c)
	}
	fmt.Println(slen)
	fmt.Println(checksum)
}

File: benchmarks/python/strings.py
def main():
    n = 100000
    token = "ab"
    s = ""
    for _ in range(n):
        s += token
    slen = len(s)
    checksum = sum(ord(c) for c in s)
    print(slen)
    print(checksum)

main()
